### PR TITLE
Add sbpy to conda-forge

### PR DESCRIPTION
**Pull request by mwcraig**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

The patch is necessary to prevent `astropy-helpers` from trying to auto-update during the build.

### CODE REVIEW

**Comment by mwcraig** [comment]
A pull request to remove the extra entry point upstream has been opened at https://github.com/NASA-Planetary-Science/sbpy/pull/267

File: recipes/sbpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.13.0
+    - astropy >=3.0
+    - matplotlib
```
Comment (scopatz) [inline recipes/sbpy/meta.yaml:27]: This should be matplotlib-base

File: recipes/sbpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - ads
+    - synphot
+    - astroquery >=0.4.1.dev0
```
Comment (scopatz) [inline recipes/sbpy/meta.yaml:30]: Can this just be `>=0.4.1`?

Comment (mwcraig) [reply recipes/sbpy/meta.yaml]: I think so, yes -- I've just updated the requirement

File: recipes/sbpy/meta.yaml (hunk 1 above)
Comment (mwcraig) [reply recipes/sbpy/meta.yaml]: This will take some getting used to 😬

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**